diff -uraN a/src/misc/picture.c b/src/misc/picture.c
--- a/src/misc/picture.c	2015-06-30 15:53:31.000000000 +0200
+++ b/src/misc/picture.c	2016-03-16 18:02:39.833522066 +0100
@@ -70,7 +70,10 @@
         i_bytes += p->i_pitch * p->i_lines;
     }
 
-    uint8_t *p_data = vlc_memalign( 16, i_bytes );
+	if((i_bytes & 31)!=0)
+    	i_bytes = (i_bytes & ~((size_t)31))+32;
+
+    uint8_t *p_data = vlc_memalign( 32, i_bytes );
     if( i_bytes > 0 && p_data == NULL )
     {
         p_pic->i_planes = 0;
@@ -173,8 +176,8 @@
     unsigned int i_ratio_h  = 1;
     for( unsigned i = 0; i < p_dsc->plane_count; i++ )
     {
-        i_modulo_w = LCM( i_modulo_w, 16 * p_dsc->p[i].w.den );
-        i_modulo_h = LCM( i_modulo_h, 16 * p_dsc->p[i].h.den );
+        i_modulo_w = LCM( i_modulo_w, 32 * p_dsc->p[i].w.den );
+        i_modulo_h = LCM( i_modulo_h, 32 * p_dsc->p[i].h.den );
         if( i_ratio_h < p_dsc->p[i].h.den )
             i_ratio_h = p_dsc->p[i].h.den;
     }
@@ -193,7 +196,7 @@
         p->i_visible_pitch = fmt->i_visible_width * p_dsc->p[i].w.num / p_dsc->p[i].w.den * p_dsc->pixel_size;
         p->i_pixel_pitch   = p_dsc->pixel_size;
 
-        assert( (p->i_pitch % 16) == 0 );
+        assert( (p->i_pitch % 32) == 0 );
     }
     p_picture->i_planes  = p_dsc->plane_count;
 
